django_celery_beat/models.py

```diff
@@ -299,6 +299,13 @@ class CrontabSchedule(models.Model):
             'Timezone to Run the Cron Schedule on. Default is UTC.'),
     )
 
+    name = models.CharField(
```

Comment:
would also like to have unit tests for new proposed changes

Reply:
I can't run any test from test_schedulers.py with the current setup. 
Will try to figure out what am I doing wrong, and possibly add some tests.